.github/PULL_REQUEST_TEMPLATE

```diff
@@ -8,3 +8,10 @@
 (If this patch involves UI changes, please attach a screenshot; otherwise, remove this)
 
 Please review http://spark.apache.org/contributing.html before opening a pull request.
+
+## Discussion detail.
```

Comment:
I don't think this will be seen at the end anyway, and this and many other details are contained in the linked doc. I realize not many people read it, but repeating it probably won't change that and we also get a weak signal about which PRs aren't serious this way.

If anything, move the "Please review..." sentence to the top, but I wouldn't bother